docs/verify/local-testing-mode.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Qiskit Runtime local testing mode
+description: Use local testing mode to simulate quantum hardware in Qiskit Runtime
+---
+
+<span id="test-locally"></span>
+# Qiskit Runtime local testing mode
```

Comment:
Is there some additional information we can add to this page about _why_ a user might want to use the local testing mode? What unique thing does it provide to users? why might they want to use this instead of just the Qiskit primitives or Qiskit Aer? Maybe @jyu00 has some suggestions for something to say on this point?